actors/builtin/miner/miner_state.go

```diff
@@ -191,12 +172,37 @@ func ConstructState(emptyArrayCid, emptyMapCid, emptyDeadlinesCid cid.Cid, owner
 	}, nil
 }
 
-func (st *State) GetWorker() addr.Address {
-	return st.Info.Worker
+func (st *State) GetInfo(store adt.Store) (*MinerInfo, error) {
+	var info MinerInfo
+	if err := store.Get(store.Context(), st.Info, &info); err != nil {
+		return nil, errors.Wrapf(err, "failed to get miner info")
+	}
+	return &info, nil
+}
+
+func (st *State) SaveInfo(store adt.Store, info *MinerInfo) error {
+	c, err := store.Put(store.Context(), info)
+	if err != nil {
+		return err
+	}
+	st.Info = c
+	return nil
+}
+
+func (st *State) GetWorker(store adt.Store) (addr.Address, error) {
```

Comment:
I think we should not have these convenience methods because they encourage an inefficient pattern of loading the miner info more than once.